Snowflake.Data/Client/SnowflakeDbConnection.cs

```diff
@@ -153,9 +158,8 @@ public override Task OpenAsync(CancellationToken cancellationToken)
                     }
                     else
                     {
-                        logger.Debug("All good");

```

Comment:
We should probably not removing this log?

Reply:
I added this mainly for myself, about 2 commits ago, so that I could do a quick grep on the log. But, you are right, it won't hurt. I'll add it back.

Reply:
Added back the line.